### ISSUE TITLE
Window Key + D that minimized window causing mouse shadow to stay in primary screen.

### ISSUE THREAD

**Issue by eternalthefirst** [PoE-TradeMacro/POE-TradeMacro#1014]
When Minimized PoE with Window key + D >> this make mouse to stay in primary screen as a shadow and be invisible in another screen. The mouse still useabe but cant see the cursor.
This only happen recently so i didn't know what happen.

AHK version: 1.1.30.03
Windows version: Window 10 64 bit 1903 (18362.239)

**Comment by Eruyome** [comment]
I don't think that this is an issue with the macro. If it where I wouldn't know how to solve it or even debug it.
